fxmanifest.lua

```diff
@@ -2,11 +2,12 @@ fx_version 'cerulean'
 game 'gta5'
 
 description 'ps-hud'
-version '2.1.0'
```

Comment:
We never really did a version, but I just released a 2.0, can you do add it as a 2.1? 

Thank you!

Reply:
Updated